packages/theme/styles/love.scss

```diff
@@ -131,3 +131,30 @@
     .floorGrid-room__field { visibility: hidden; }
   }
 }
+
+// RoomPopup
```

Comment:
These changes does not seem to be related to the typo fix. Could you remove them?

Reply:
yeah removed. Not sure where that came from. Sorry about that